### PR TITLE
Integration branch

### PR DESCRIPTION
**Pull request by omk963**
Merged the HomePage, RecipePage, RecipeCard (Added to FavoritesPage)

### CODE REVIEW

File: src/components/FavoritesPage/FavoritesPage.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import RecipeList from '../RecipeList';
+import { recipes } from '../../list'; // stand-in data until we have access to MongoDB
+
+const FavoritesPage = () => {
+    return (
+        <>
+            <h1>Hearts</h1>
```
Comment (v-sevilla) [inline src/components/FavoritesPage/FavoritesPage.jsx]: Change the title to favorites!

File: src/components/RecipePage/RecipePage.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      <RecipeHeader title={recipe.title} cookTime={recipe.cookTime} />
+      <RecipeImage image={recipe.image} />
+      <RecipeContent ingredients={recipe.ingredients} steps={recipe.steps} />
+    </div>
+  );
+};
+
+export default RecipePage;
+
+// //
```
Comment (v-sevilla) [inline src/components/RecipePage/RecipePage.jsx]: We can remove the commented-out component tree.

**Review by viktorijakm — APPROVED** [review]

**Comment by v-sevilla** [comment]
Looks good on my end, everything is functioning properly 👍🏻

**Review by v-sevilla — APPROVED** [review]
Looks good on my end, everything is functioning properly 👍🏻

**Pull request merged**